tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
Everything should be movable (unless there's a _really_ good reason for an exception).  I strongly disagree that that implies everything should have equality operators.  There are classes that you just shouldn't have any reason to compare (e.g., Options), and others where the meaning of equality is unclear (e.g., smart pointers).